examples/restricted-assets/assets/poi-approval.teal

```diff
@@ -227,7 +229,7 @@ bz failed
 // tx 1 - clawback transactions that moves the frozen asset from sender to receiver - signed by clawback-escrow
 // verify the account sent in the accounts array is
 // Note: the receiver of the asset is asset xfer
-gtxn 0 Accounts 1
+gtxna 0 Accounts 1
```

Comment:
we have support for array in gtxna. right ?

Reply:
or we will update it after fix gets merged ?

Reply:
Yes, we already had support for array in `gtxna`. After fix we will have support for array with `txn`, `gtxn` as well. 
Updated TEAL code here because i think `gtxna 0 Accounts 1` is more intuitive than `gtxn 0 Accounts 1`. Both will work after the fix. Let me know what you think we should keep here